remove-node.yml

```diff
@@ -18,12 +18,15 @@
 
 - hosts: kube-master
   roles:
+    - { role: kubespray-defaults }
```

Comment:
delete it. Only need it when reset  node.

Reply:
I have deleted this line in my new commit. Kindly review it.